### PR TITLE
[Dev docs] [update-content] [1477635]

### PR DESCRIPTION
**Pull request by Mahesh2516**
Updated the content and manifest schema.

### CODE REVIEW

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Upload your custom app
 description: Describes how to upload your app in Microsoft Teams
 ms.topic: how-to
+ms.author:
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/apps-upload.md:5]: Do not keep the field empty. Add La's name here. lajanuar

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md [hunk 2]
```diff
@@ -103,13 +104,28 @@ With the app loaded in the team or conversation, users can set up a Connector us
...
-## Removing or updating your app
+## Add an app
+
+Although installing an app in the personal scope works for most apps, some of the apps in Teams Store support both personal and team scopes.
+Some of these apps are intended to work in a team or a group chat, with personal app experience being secondary.
+Hence, the default install scope selection helps you to specify the `defaultInstallScope` for the apps that you publish. The app installation experience makes the default options available to the user, while the rest is moved under the chevron as highlighted in the image.
+
+![Add an app](../../assets/images/compose-extensions/addanapp.png)
+
+In the specified app manifest, the property `defaultInstallScope` supports values like personal, team, group chat, or meeting.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/apps-upload.md:121]: What is 'specified' app manifest? Where is is 'specified'?

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md [hunk 3]
```diff
@@ -103,13 +104,28 @@ With the app loaded in the team or conversation, users can set up a Connector us
...
+
+> [!NOTE]
+> Ensure your app specifies a `defaultInstallScope` that is supported by all the capabilities in the app for the validation to pass. In other words, every single capability in an app must support the `defaultInstallScope`. Otherwise, the behavior reverts to the experience where no default installation scope is provided.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/apps-upload.md:124]: Reword and make it precise. The intent of the note is not coming out.

File: msteams-platform/resources/schema/manifest-schema.md [hunk 4]
```diff
@@ -271,45 +270,95 @@ The following schema sample shows all extensibility options.
...
+      }
+    },
+    "description": "When a group install scope is selected, this defines the default capability when the user installs the app.",
+    "additionalProperties": false
+
 }
 ```
 
+> [!Note]
+> For an updated version of the schema, see also: https://developer.microsoft.com/json-schemas/teams/v1.9/MicrosoftTeams.schema.json.
```
Comment (KirtiPereira) [inline msteams-platform/resources/schema/manifest-schema.md:322]:
```suggestion
> For an updated version of the schema, see [MicrosoftTeams.schema.json](https://developer.microsoft.com/json-schemas/teams/v1.9/MicrosoftTeams.schema.json).
```

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 - verify and incorporate the comments and do a thorough self-review.

File: msteams-platform/resources/schema/manifest-schema.md [hunk 5]
```diff
@@ -271,50 +270,99 @@ The following schema sample shows all extensibility options.
...
+Optional, but recommended — string
 
 The https:// URL referencing the JSON Schema for the manifest.
 
 ## manifestVersion
 
 **Required** — string
 
-The version of the manifest schema this manifest is using. It should be "1.7".
+The version of manifest schema this manifest is using. It must be "1.7".
```
Comment (KirtiPereira) [inline msteams-platform/resources/schema/manifest-schema.md:335]: Please correct the markdown format for 1.7. Why the double quotes?

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md [hunk 6]
```diff
@@ -41,83 +42,98 @@ There are two ways to upload your app to Teams:
...
+The `defaultInstallScope` property supports values, such as personal, team, group chat, or meeting.
+
+> [!NOTE]
+> For a successful validation, specify a `defaultInstallScope` that supports all capabilities. Each capability in an app must support the `defaultInstallScope`, else the app acts as it would without the default installation scope.
+
+`defaultGroupCapability` defines capabilities that are installed by default. Choose tab, bot, or connector as the default capability of the app for each group scope. The app validation fails if any of the scopes do not support the default capability.
+
+## Remove or update your app
+
+To remove your app, select the delete icon next to the app name in the View Teams bots list.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/apps-upload.md:130]: Check the formatting.

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md [hunk 7]
```diff
@@ -41,83 +42,98 @@ There are two ways to upload your app to Teams:
...
+
+If you change manifest information, you must first remove the app and then add the updated package per [Load your package into a team](#load-your-package-into-teams). Code changes on your service do not require you to re-upload your manifest, unless those changes require manifest updates, such as changes to the URL or the Microsoft app ID for its bot.
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/apps-upload.md:132]: Check if we can use **re-upload** as per MSTP. 
```suggestion
If you change manifest information, first remove the app and then add the updated package, see [Load your package into a team](#load-your-package-into-teams). Code changes on your service do not require you to upload your manifest again. However, if the code changes require manifest updates, such as changes to the URL or the Microsoft app ID for its bot, you must upload the manifest again.
```

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md [hunk 8]
```diff
@@ -41,83 +42,98 @@ There are two ways to upload your app to Teams:
...
 
 > [!NOTE]
-> There is no way to completely remove a bot from personal context. If the bot is removed and re-added, additional communication with the bot will append to the previous conversation.
+> You cannot remove a bot from a personal context entirely. If the bot is removed and added again, additional communication with the bot appends to the previous conversation.
 
 ## Troubleshooting notes
 
-* If the manifest doesn't load, please double-check that you followed all the instructions in [Create the package](../../concepts/build-and-test/apps-package.md) and validated your manifest against the [schema](../../resources/schema/manifest-schema.md).
+If the manifest fails to load, check that you have followed all the instructions in [Create the package](../../concepts/build-and-test/apps-package.md) and validated your manifest against the [schema](../../resources/schema/manifest-schema.md).
```
Comment (KirtiPereira) [inline msteams-platform/concepts/deploy-and-publish/apps-upload.md:139]:
```suggestion
If the manifest fails to load, check if you have followed all the instructions in [Create the package](../../concepts/build-and-test/apps-package.md) and validated your manifest against the [schema](../../resources/schema/manifest-schema.md).
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 9]
```diff
@@ -1,15 +1,14 @@
 ---
 title: Manifest schema reference
-description: Describes the manifest schema for Microsoft Teams
+description: describes the manifest schema for Microsoft Teams
```
Comment (KirtiPereira) [inline msteams-platform/resources/schema/manifest-schema.md:3]: Why did you change the capitalization.

File: msteams-platform/resources/schema/manifest-schema.md [hunk 10]
```diff
@@ -271,50 +270,99 @@ The following schema sample shows all extensibility options.
...
 
 This version string must follow the [semver](http://semver.org/) standard (MAJOR.MINOR.PATCH).
 
 ## id
 
 **Required** — Microsoft app ID
 
-The unique Microsoft-generated identifier for this app. If you have registered a bot via the Microsoft Bot Framework, or your tab's web app already signs in with Microsoft, you should already have an ID and should enter it here. Otherwise, you should generate a new ID at the Microsoft Application Registration Portal ([My Applications](https://apps.dev.microsoft.com)), enter it here, and then reuse it when you add a bot.Note: If you are submitting an update to your existing app in AppSource, the ID in your manifest must not be modified.
+The unique Microsoft-generated identifier for this app. If your bot is registered via the Microsoft Bot Framework or your tab's web app already signs in with Microsoft, you already have an ID and must enter it here. Otherwise, you must generate a new ID at the Microsoft Application Registration Portal ([My Applications](https://apps.dev.microsoft.com)), enter it here, and then reuse it when you add a bot.
```
Comment (KirtiPereira) [inline msteams-platform/resources/schema/manifest-schema.md:351]: Please rework on this paragraph. Follow MSTP conventions.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@Mahesh2516 - pelase verify and commit the changes. Also, please do a thorough self-review of manifest-schema file.

File: msteams-platform/concepts/deploy-and-publish/apps-upload.md (hunk 6 above)
Comment (Mahesh2516) [reply msteams-platform/concepts/deploy-and-publish/apps-upload.md]: corrected

**Comment by Mahesh2516** [comment]
Hi @matrive-msft, kindly share your comments or approve.

**Review by matrive-msft — DISMISSED** [review]

**Review by KirtiPereira — DISMISSED** [review]
Approved.

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Review by KirtiPereira — APPROVED** [review]
Approved
